README.md

```diff
@@ -29,3 +31,8 @@ Contributors:
 Contributing Breakers:
```

Comment:
@mbjoseph  will you kindly add yourself as a contributor here? 
Perhaps we should just have a contributor list and include Chris H and me :) i'm trying not to be a breaker these days!

Reply:
Sure thing! I agree it's probably better not to split hairs between contributors and breakers :)

Reply:
Fixed with https://github.com/earthlab/earthpy/pull/18/commits/c4f9cf5059b3c630c8470d94601ac0f4b8858ab8